automation/lights.yaml

```diff
@@ -16,3 +16,79 @@
   action:
   - service: light.turn_off
     entity_id: light.linear_lb60z1_dimmable_led_light_bulb_level
+
+
+- id: entry_lights_on_after_unlock_day
+  alias: Turn entry lights on when unlocked day
+  trigger:
+    - platform: state
+      entity_id: lock.schlage_be469_touchscreen_deadbolt_locked_3
```

Comment:
eventually will want to get this triggering off of only exterior unlocking, but this is fine for now.